ecl/ecl-package/CMakeLists.txt

```diff
@@ -56,7 +56,12 @@ include_directories (
 ADD_DEFINITIONS( -D_CONSOLE )
 
 HPCC_ADD_EXECUTABLE ( ecl-packagemap ${SRCS} )
-add_dependencies ( ecl-packagemap espscm ws_packageprocess )
+
+set( WS_PACKAGEPROCESS_DEP )
+if( NOT CLIENTTOOLS_ONLY )
```

Comment:
Builds are OK with old cmake 2.8.11+.

Currently ws_packageprocess and ws_workunits are only build with "Platform" which is why there are not available in "Clienttools-Only" build.

Guess I should make ws_packageprocess and ws_workunits build on "Clienttools" which is enabled with "Platform" build.